services/movie.py

```diff
@@ -0,0 +1,33 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
+               ) -> QuerySet:
+    movies = Movie.objects.all()
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no Movie with the given ID is found. Consider handling this exception or using `get_object_or_404` if this function is used in a view to provide a more user-friendly error handling.